### DISCUSSION TITLE
vercel build error  TypeError: f.charCodeAt is not a function

### DISCUSSION THREAD

**Discussion by sahinme** [vercel/next.js#15537] [Help]
Hi ! I have create-next-app and in production get error like this :  TypeError: f.charCodeAt is not a function.  I using vercel host.
It is working fine in localhost only get error while deploying. Pls help !
![err](https://user-images.githubusercontent.com/40808896/88588664-998fd700-d060-11ea-95ea-85a46f9f8b28.png)

**Comment by alanhoskins** [comment]
I've been hitting the same error today with Vercel. It was working fine locally until I blew away my node_modules and reinstalled. Now even locally it won't build.

**Reply by sahinme** [reply]
This build error appeared after I added txt file to public folder. Then i removed changes but it still dont build

**Reply by alanhoskins** [reply]
Appears to be some package dependencies that are causing issues. After some digging I found adding 

```
  "resolutions": {
    "cssnano-preset-simple": "1.1.1",
    "webpack": "4.43"
  }
```

to my package dependencies, it works. You may not need the webpack one, but I did for Amplify. 

Hope it helps to temp fix things for you.

[Amplify issue](https://github.com/aws-amplify/amplify-js/issues/6411)
[cssnano issue](https://github.com/Timer/cssnano-preset-simple/issues/6)

**Reply by sahinme** [reply]
Thank you @alanhoskins ! I solved problem. I was added yesterday dotenv-webpack module. I dont use it and removed. Now it is working fine.

**Comment by Timer** [comment]
Duplicate of https://github.com/vercel/next.js/issues/15026
